### PR TITLE
Delete style

### PR DESCRIPTION
**Pull request by firmusoft**
Copy paste of paragraph with style not working

### CODE REVIEW

File: src/ui/CustomMenuUI.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import * as React from 'react';
+import  * as React from 'react';
```
Comment (Tezra) [inline src/ui/CustomMenuUI.js:1]: extra space

File: src/CustomStyleCommand.js [hunk 2]
```diff
@@ -529,6 +528,14 @@ function compareMarkWithStyle(
       case MARK_TEXT_COLOR:
         same = mark.attrs['color'] === style[COLOR];
         break;
+      case MARK_TEXT_HIGHLIGHT:
```
Comment (Tezra) [inline src/CustomStyleCommand.js:531]: MARK_TEXT_HIGHLIGHT is already in this switch. Remove the duplicate case.

File: src/CustomStyleCommand.js [hunk 3]
```diff
@@ -1318,9 +1335,11 @@ export function applyLatestStyle(
   node: Node,
```
Comment (Tezra) [inline src/CustomStyleCommand.js:1335]: Why aren't these typescript files?

**Review by Tezra — CHANGES_REQUESTED** [review]
Several things that need to be cleaned up as well as a couple questions.

**Review by Tezra — APPROVED** [review]

**Review by Tezra — APPROVED** [review]

**Review by rodgersgb — APPROVED** [review]

**Pull request merged**